src/card/card.jsx

```diff
@@ -12,12 +12,46 @@ const Card = React.createClass({
     };
   },
 
+  getDefaultProps() {
```

Comment:
Should I make a PR to enforce `react/sort-comp: 0 # Wishlist, one day` :angel: ?